common/test/test_backintime.py

```diff
@@ -141,12 +141,8 @@ def test_local_snapshot_is_successful(self):
         #       The same goes with Gtk warnings.
 
         line_beginnings_to_exclude = [
-            "WARNING: Failed to connect to Udev serviceHelper",
-            "WARNING: D-Bus message:",
-            "WARNING: Udev-based profiles cannot be changed or checked",
-            "WARNING: Inhibit Suspend failed",
-            "Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway",
```

Comment:
Should we keep the well-known warnings to be ignored in a comment at least?

Reply:
Good idea. Added them as comment.